main.py

```diff
@@ -1,7 +1,31 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' should be corrected to 'Drama' to match the update operation on line 21.